chapters/stack/functions/drills/tasks/print_rev_string/solution/print_reverse_string.asm

```diff
@@ -0,0 +1,71 @@
+section .data
```

Comment:
Maybe add some explanation comments if this is the solution. I think students should read the comments rather than copy the code